web/modules/custom/usagov_wizard/usagov_wizard.libraries.yml

```diff
@@ -0,0 +1,6 @@
+expand_collapse:
+  js:
+    js/expand_collapse.js: {}
+  dependencies:
+    - core/jquery
+    - core/once
```

Comment:
you should list jstree plugin as a dependency

Reply:
Good point, just added it.